src/consts.rs

```diff
@@ -173,90 +171,6 @@ impl PartialOrd for Constant {
     }
 }
 
-fn format_byte(fmt: &mut fmt::Formatter, b: u8) -> fmt::Result {
-    if b == b'\\' {
-        write!(fmt, "\\\\")
-    } else if 0x20 <= b && b <= 0x7e {
-        write!(fmt, "{}", char::from_u32(b as u32).expect("all u8 are valid char"))
-    } else if b == 0x0a {
-        write!(fmt, "\\n")
-    } else if b == 0x0d {
-        write!(fmt, "\\r")
-    } else {
-        write!(fmt, "\\x{:02x}", b)
-    }
-}
-
-impl fmt::Display for Constant {
```

Comment:
Is there a specific reason for removing this (apart from it using Debug formatting to write out `Str`s)?